scripts/setup-macos.sh

```diff
@@ -195,6 +134,7 @@ function install_velox_deps {
   run_and_time install_wangle
```

Comment:
As mentioned in centos I think thrift and arrow should be in common and also be installed here

Reply:
Ok. I will add them here. The current macos.sh script doesn't install arrow and thrift so I thought those won't be needed here.